TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/BotActivity.java

```diff
@@ -78,6 +78,13 @@ public void onLazersReleased() {
         }
     };
 
+    private final ServerDeclarationListener serverDeclarationListener = new ServerDeclarationListener() {
```

Comment:
We have a man in the middle server. I could probably set it up so that one is the server and one is the client but I haven't looked into it yet. Would rather have this change in, as I know it works and work on having one server one client.